client/src/App.jsx

```diff
@@ -15,8 +15,13 @@ class App extends React.Component {
     super(props);
     this.state = {
       product_Id: 40358,
-      product_name: null
+      product_name: 'Camo Onesie'
     };
+    this.handleProductNameChange = this.handleProductNameChange.bind(this);
+  }
+
+  handleProductNameChange (name) {
+    this.setState({product_name: name});
```

Comment:
saw your Slack comment on this and will update my Overview component after this is approved.

Reply:
dooope! thanks Daniel!